lighthouse-cli/bin.ts

```diff
@@ -114,6 +116,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
   .default('port', 9222)
+  .default('timeout', 25)
```

Comment:
I think in general (at least from a documentation perspective) it's better to make sure they're the same, but in this case my main problem would be development in driver, where you'd have to remember why changes to `MAX_WAIT_FOR_FULLY_LOADED` aren't doing anything when the CLI is run.